### ISSUE TITLE
The #[time] macro seems to strip the "unsafe" keyword from a decorated function

### ISSUE THREAD

**Issue by DavidAntliff** [PhilipDaniels/logging_timer#4]
If I write an unsafe function and apply the `#[time]` decorator macro:

```
#[time]
pub unsafe fn do_something_risky() { ... }
```

I am now able to call this function outside of an unsafe block, e.g.

```
fn main() {
    do_something_risky();
}
```

Whereas I would expect to have to write:

```
fn main() {
    unsafe { 
        do_something_risky();
    }
}
```

If `do_something_risky()` needs to be `unsafe` to call other unsafe functions, then fortunately this generates a compiler error, but for other unsafe behaviour that the compiler can't know about, for example multi-threading invariants documented by the programmer, this is pretty serious, because there's currently no way for the programmer to _require_ `unsafe` when calling such a decorated function.

A workaround is to create an unsafe wrapper function, so that `#[time]` is only on the inner function, but this is quite messy.

**Comment by PhilipDaniels** [comment]
Fixed in v1.1.1.

**Comment by DavidAntliff** [comment]
Thank you, seems to be working correctly for me now.
